schemas/RecipeProcess.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/RecipeProcess.json",
+    "title": "RecipeProcess",
+    "description": "Specifies the process part of a recipe for use in planning from recipe.",
```

Comment:
How about
"Specifies a process in a recipe for use in planning from recipe.",

Reply:
> How about
> "Specifies a process in a recipe for use in planning from recipe."

As of now, it talks about a "process part". And yes I know you meant "part of a recipe", but this reads like a recipe has something called a process part. Like as opposed to the resource part.